### ISSUE TITLE
Saving Scatterers removes tied parameters

### ISSUE THREAD

**Issue by barkls** [manoharan-lab/holopy#257]
This only occurs when the parameter in question has a long string of digits so it may be a rounding error. It is a problem when doing fits since the `Model` object's parameters don't match the `FitResult` object's parameters.

Reproduce with the following code:
```
import numpy
from holopy.inference import prior
from holopy.scattering import Sphere, Spheres
n = prior.Uniform(1,2)
r = prior.Uniform(0, np.pi/2)
s1 = Spheres([Sphere(p) for i in range(2)])
print(s1.parameters)
hp.save('test.out',s1)
s2 = hp.load('test.out')
print(s2.parameters)
```

note that `n` remains tied but `r` does not. Possible solutions are to 
1) round values upon defining a Scatterers object, 
2) have scatterers take a "tied" argument that can be specified and stored
3) customize saving/loading of Scatterers objects

**Comment by barkls** [comment]
This bug means that some methods don't work in a saved FitResult object, such as max_lnprob

**Comment by briandleahy** [comment]
Fixed by #290
